dxr/build.py

```diff
@@ -342,7 +342,7 @@ def ensure_folder(folder, clean=False):
     if clean and os.path.isdir(folder):
         shutil.rmtree(folder, False)
     if not os.path.isdir(folder):
-        mkdir(folder)
+        os.makedirs(folder)
```

Comment:
I think I ran into this back when I was copying binary files/images to a separate target directory. It might not be necessary anymore.